api/src/main/resources/messages.properties

```diff
@@ -54,7 +54,7 @@ pihcore.visitNote.visitTemplate=Visit Template
 pihcore.visitNote.clinicalNotes=Clinical Notes
 pihcore.visitNote.duringCurrentConsult=During current consult
 
-pihcore.chiefComplaint.title=Chief Complaint
```

Comment:
Why remove this? I believe we use it elsewhere....

Reply:
Oh shoot, good catch!